src/app/homepage/menu/menu.component.ts

```diff
@@ -240,6 +240,7 @@ export class MenuComponent implements OnInit {
         { title: 'Prisma', path: '/recipes/prisma' },
         { title: 'Serve static', path: '/recipes/serve-static' },
         { title: 'Commander', path: '/recipes/nest-commander' },
+        { title: `Nestia`, path: `/recipes/nestia` },
```

Comment:
```suggestion
        { title: `Nestia (code generator)`, path: `/recipes/nestia` },
```

or something like that